src/sssom/util.py

```diff
@@ -109,6 +109,25 @@ def clean_context(self) -> None:
         """Clean up the context."""
         self.converter = curies.chain([_get_built_in_prefix_map(), self.converter])
 
+    def standardize_references(self) -> None:
+        """Standardize this MSDF's dataframe and metadata with respect to its converter."""
+        self._standardize_metadata_references()
+        self._standardize_df_references()
+
+    def _standardize_df_references(self) -> None:
+        """Standardize this MSDF's dataframe with respect to its converter."""
+        func = partial(_standardize_curie_or_iri, converter=self.converter)
```

Comment:
What on earth is a `partial()` is this like black magic?

Reply:
no way, `partial` is like one of the best tools in Python. It makes working with functions where you want to keep something constant much easier. This is similar to `func = lambda curie_or_iri: _standardize_curie_or_iri(curie_or_iri, converter=self.converter)`

Reply:
Nice!